stories/topics/con-gcp-use-container-image.adoc

```diff
@@ -1,10 +1,23 @@
 [id="con-gcp-use-container-image"]
 
-= Using the ansible-on-clouds-ops container image
+= Pulling the ansible-on-clouds-ops container image
+
+Pull the Docker image for the Ansible on Clouds operational container which aligns with the version of your foundation deployment.
+
+For example, if your foundation deployment version is 2.3.20230221-00, you must pull the operational image with tag 2.3.20230221.
```

Comment:
Again the -00?

Reply:
The tag used to pull will not have -00 (due to downstream build publication limitations in the tooling)